src/main/java/com/trashit/back/services/FileService.java

```diff
@@ -24,27 +27,55 @@
     private Path fileStorageLocation;
 
     public String uploadFile(MultipartFile file) {
-        Path newLocation = Paths.get(String.valueOf(fileStorageLocation)).toAbsolutePath().normalize();
         try {
-            Files.createDirectories(newLocation);
-        } catch (Exception ex) {
-            throw new FileException("Could not create the directory where the uploaded files will be stored.", ex);
+            if (file.isEmpty() || file == null) {
```

Comment:
```suggestion
            if (file.isEmpty()) {
```
`File` sera toujours `null` vu qu'on appelle cette fontion que si `file` est pas `null` :wink: